branca/element.py

```diff
@@ -215,47 +215,13 @@ def __init__(self, url, download=False):
         if download:
             self.get_code()
 
-        self._template = Template(u"""
-        {% if kwargs.get("embedded",False) %}
-            <style>{{this.get_code()}}</style>
-        {% else %}
-            <link rel="stylesheet" href="{{this.url}}" />
-        {% endif %}
-        """)
-
-_default_js = [
-    ('leaflet',
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"),
-    ('jquery',
-     "https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"),
-    ('bootstrap',
-     "https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"),
-    ('awesome_markers',
-     "https://rawgithub.com/lvoogdt/Leaflet.awesome-markers/2.0/develop/dist/leaflet.awesome-markers.js"),  # noqa
-    ('marker_cluster_src',
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.4.0/leaflet.markercluster-src.js"),  # noqa
-    ('marker_cluster',
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.4.0/leaflet.markercluster.js"),  # noqa
-    ]
-
-_default_css = [
-    ("leaflet_css",
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.css"),
-    ("bootstrap_css",
-     "https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css"),
-    ("bootstrap_theme_css",
-     "https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css"),  # noqa
-    ("awesome_markers_font_css",
-     "https://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css"),  # noqa
-    ("awesome_markers_css",
-     "https://rawgit.com/lvoogdt/Leaflet.awesome-markers/2.0/develop/dist/leaflet.awesome-markers.css"),  # noqa
-    ("marker_cluster_default_css",
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.4.0/MarkerCluster.Default.css"),  # noqa
-    ("marker_cluster_css",
-     "https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.4.0/MarkerCluster.css"),  # noqa
-    ("awesome_rotate_css",
-     "https://raw.githubusercontent.com/python-visualization/folium/master/folium/templates/leaflet.awesome.rotate.css"),  # noqa
-    ]
```

Comment:
I need to get familiar with branca to make meaningful comments. So I have more questions than "reviews."

Are you removing the default JS and CSS because they belong in folium?